app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/search_box.scala.html

```diff
@@ -21,7 +21,7 @@
 @helper.form(action = controllers.routes.SearchController.search()) {
     <label for="search_box-input" class="form-label">@label</label>
     <div class="search_box @containerClass">
-        <input tabindex="100" id="search_box-input" class="form-control" name="reference" value="" type="text" placeholder="Enter full case reference">
+        <input tabindex="100" id="search_box-input" class="form-control" name="reference" value="" type="text">
         <button tabindex="101" aria-label="search for case reference" id="search_box-button" type="submit"></button>
```

Comment:
can we use messages to get the messages form file?

Reply:
This has now been addressed.